Tests fail with docutils 0.17.1 on Fedora Rawhide

### Describe the problem

The following tests fail when building with docutils 0.17.1.
```
FAILED tests/test_build.py::test_conditional_assets_html_assets_policy[index]
FAILED tests/test_build.py::test_conditional_assets_html_assets_policy[no_tabs1]
FAILED tests/test_build.py::test_conditional_assets_html_assets_policy[no_tabs2]
```


### Link to your repository or website

https://bugzilla.redhat.com/show_bug.cgi?id=2005976

### Steps to reproduce

1. Update docutils from 0.16 to 0.17.1
2. Attempt package rebuild
3. pytest produces failed tests.

### The version of Python you're using

3.10

### Your operating system

Fedora Rawhide x86_64

### Versions of your packages

```
python-setuptools-wheel-57.4.0-1.fc35.noarch
python-pip-wheel-21.2.3-2.fc36.noarch
python3-3.10.0~rc2-2.fc36.x86_64
python3-libs-3.10.0~rc2-2.fc36.x86_64
python3-setuptools-57.4.0-1.fc35.noarch
python3-idna-3.2-1.fc35.noarch
python3-py-1.10.0-5.fc35.noarch
python3-attrs-21.2.0-4.fc35.noarch
python3-charset-normalizer-2.0.6-1.fc36.noarch
python3-imagesize-1.2.0-6.fc35.noarch
python3-iniconfig-1.1.1-5.fc35.noarch
python3-markupsafe-2.0.0-2.fc35.x86_64
python3-pluggy-1.0.0-1.fc36.noarch
python3-pygments-2.10.0-1.fc36.noarch
python3-pyparsing-2.4.7-9.fc35.noarch
python3-packaging-21.0-2.fc35.noarch
python3-rpm-generators-12-8.fc35.noarch
python3-pysocks-1.7.1-11.fc35.noarch
python3-pytz-2021.1-4.fc35.noarch
python3-babel-2.9.1-4.fc35.noarch
python3-jinja2-3.0.1-2.fc35.noarch
python3-pyyaml-5.4.1-4.fc35.x86_64
python3-six-1.16.0-4.fc35.noarch
python3-urllib3-1.26.6-2.fc35.noarch
python3-requests-2.26.0-1.fc35.noarch
python3-snowballstemmer-1.9.0-10.fc35.noarch
python3-soupsieve-2.2.1-2.fc35.noarch
python3-lxml-4.6.3-4.fc35.x86_64
python3-beautifulsoup4-4.10.0-1.fc36.noarch
python3-sphinx-theme-alabaster-0.7.12-13.fc35.noarch
python3-sphinxcontrib-applehelp-1.0.2-6.fc35.noarch
python3-sphinxcontrib-devhelp-1.0.2-6.fc35.noarch
python3-sphinxcontrib-htmlhelp-2.0.0-2.fc35.noarch
python3-sphinxcontrib-jsmath-1.0.1-13.fc35.noarch
python3-sphinxcontrib-qthelp-1.0.3-6.fc35.noarch
python3-sphinxcontrib-serializinghtml-1.1.5-2.fc35.noarch
python3-toml-0.10.2-5.fc35.noarch
python3-pytest-6.2.5-1.fc36.noarch
python3-pytest-datadir-1.3.1-1.fc35.noarch
python3-docutils-0.17.1-1.fc36.noarch
python3-sphinx-4.2.0-1.fc36.noarch
python3-pytest-regressions-2.2.0-1.fc35.noarch
python3-devel-3.10.0~rc2-2.fc36.x86_64
python3-wheel-0.37.0-1.fc36.noarch
python3-pip-21.2.3-2.fc36.noarch
```

### Additional context

_No response_

--- a/setup.py
+++ b/setup.py
@@ -25,7 +25,7 @@ setup(
     url="https://github.com/executablebooks/sphinx-tabs",
     license="MIT",
     python_requires="~=3.6",
-    install_requires=["sphinx>=2,<5", "pygments", "docutils~=0.16.0"],
+    install_requires=["sphinx>=2,<5", "pygments", "docutils~=0.17.0"],
     extras_require={
         "testing": [
             "coverage",

--- a/tests/test_build/test_conditional_assets_html_assets_policy_index_.html
+++ b/tests/test_build/test_conditional_assets_html_assets_policy_index_.html
@@ -15,7 +15,7 @@
      </li>
     </ul>
    </div>
-   <div class="section" id="fruits">
+   <section id="fruits">
     <h1>
      Fruits
      <a class="headerlink" href="#fruits" title="Permalink to this headline">
@@ -50,8 +50,8 @@
       </p>
      </div>
     </div>
-   </div>
-   <div class="section" id="luminaries">
+   </section>
+   <section id="luminaries">
     <h1>
      Luminaries
      <a class="headerlink" href="#luminaries" title="Permalink to this headline">
@@ -78,8 +78,8 @@
       </p>
      </div>
     </div>
-   </div>
-   <div class="section" id="code-tabs">
+   </section>
+   <section id="code-tabs">
     <h1>
      Code Tabs
      <a class="headerlink" href="#code-tabs" title="Permalink to this headline">
@@ -178,8 +178,8 @@
       </div>
      </div>
     </div>
-   </div>
-   <div class="section" id="group-tabs">
+   </section>
+   <section id="group-tabs">
     <h1>
      Group Tabs
      <a class="headerlink" href="#group-tabs" title="Permalink to this headline">
@@ -242,7 +242,7 @@
       </p>
      </div>
     </div>
-   </div>
+   </section>
   </div>
  </div>
 </div>

--- a/tests/test_build/test_conditional_assets_html_assets_policy_no_tabs1_.html
+++ b/tests/test_build/test_conditional_assets_html_assets_policy_no_tabs1_.html
@@ -15,7 +15,7 @@
      </li>
     </ul>
    </div>
-   <div class="section" id="fruits">
+   <section id="fruits">
     <h1>
      Fruits
      <a class="headerlink" href="#fruits" title="Permalink to this headline">
@@ -50,8 +50,8 @@
       </p>
      </div>
     </div>
-   </div>
-   <div class="section" id="luminaries">
+   </section>
+   <section id="luminaries">
     <h1>
      Luminaries
      <a class="headerlink" href="#luminaries" title="Permalink to this headline">
@@ -78,8 +78,8 @@
       </p>
      </div>
     </div>
-   </div>
-   <div class="section" id="code-tabs">
+   </section>
+   <section id="code-tabs">
     <h1>
      Code Tabs
      <a class="headerlink" href="#code-tabs" title="Permalink to this headline">
@@ -178,8 +178,8 @@
       </div>
      </div>
     </div>
-   </div>
-   <div class="section" id="group-tabs">
+   </section>
+   <section id="group-tabs">
     <h1>
      Group Tabs
      <a class="headerlink" href="#group-tabs" title="Permalink to this headline">
@@ -242,7 +242,7 @@
       </p>
      </div>
     </div>
-   </div>
+   </section>
   </div>
  </div>
 </div>

--- a/tests/test_build/test_conditional_assets_html_assets_policy_no_tabs2_.html
+++ b/tests/test_build/test_conditional_assets_html_assets_policy_no_tabs2_.html
@@ -15,7 +15,7 @@
      </li>
     </ul>
    </div>
-   <div class="section" id="fruits">
+   <section id="fruits">
     <h1>
      Fruits
      <a class="headerlink" href="#fruits" title="Permalink to this headline">
@@ -50,8 +50,8 @@
       </p>
      </div>
     </div>
-   </div>
-   <div class="section" id="luminaries">
+   </section>
+   <section id="luminaries">
     <h1>
      Luminaries
      <a class="headerlink" href="#luminaries" title="Permalink to this headline">
@@ -78,8 +78,8 @@
       </p>
      </div>
     </div>
-   </div>
-   <div class="section" id="code-tabs">
+   </section>
+   <section id="code-tabs">
     <h1>
      Code Tabs
      <a class="headerlink" href="#code-tabs" title="Permalink to this headline">
@@ -178,8 +178,8 @@
       </div>
      </div>
     </div>
-   </div>
-   <div class="section" id="group-tabs">
+   </section>
+   <section id="group-tabs">
     <h1>
      Group Tabs
      <a class="headerlink" href="#group-tabs" title="Permalink to this headline">
@@ -242,7 +242,7 @@
       </p>
      </div>
     </div>
-   </div>
+   </section>
   </div>
  </div>
 </div>

--- a/tests/test_build/test_conditional_assets_index_.html
+++ b/tests/test_build/test_conditional_assets_index_.html
@@ -15,7 +15,7 @@
      </li>
     </ul>
    </div>
-   <div class="section" id="fruits">
+   <section id="fruits">
     <h1>
      Fruits
      <a class="headerlink" href="#fruits" title="Permalink to this headline">
@@ -50,8 +50,8 @@
       </p>
      </div>
     </div>
-   </div>
-   <div class="section" id="luminaries">
+   </section>
+   <section id="luminaries">
     <h1>
      Luminaries
      <a class="headerlink" href="#luminaries" title="Permalink to this headline">
@@ -78,8 +78,8 @@
       </p>
      </div>
     </div>
-   </div>
-   <div class="section" id="code-tabs">
+   </section>
+   <section id="code-tabs">
     <h1>
      Code Tabs
      <a class="headerlink" href="#code-tabs" title="Permalink to this headline">
@@ -178,8 +178,8 @@
       </div>
      </div>
     </div>
-   </div>
-   <div class="section" id="group-tabs">
+   </section>
+   <section id="group-tabs">
     <h1>
      Group Tabs
      <a class="headerlink" href="#group-tabs" title="Permalink to this headline">
@@ -242,7 +242,7 @@
       </p>
      </div>
     </div>
-   </div>
+   </section>
   </div>
  </div>
 </div>

--- a/tests/test_build/test_conditional_assets_no_tabs1_.html
+++ b/tests/test_build/test_conditional_assets_no_tabs1_.html
@@ -15,7 +15,7 @@
      </li>
     </ul>
    </div>
-   <div class="section" id="fruits">
+   <section id="fruits">
     <h1>
      Fruits
      <a class="headerlink" href="#fruits" title="Permalink to this headline">
@@ -50,8 +50,8 @@
       </p>
      </div>
     </div>
-   </div>
-   <div class="section" id="luminaries">
+   </section>
+   <section id="luminaries">
     <h1>
      Luminaries
      <a class="headerlink" href="#luminaries" title="Permalink to this headline">
@@ -78,8 +78,8 @@
       </p>
      </div>
     </div>
-   </div>
-   <div class="section" id="code-tabs">
+   </section>
+   <section id="code-tabs">
     <h1>
      Code Tabs
      <a class="headerlink" href="#code-tabs" title="Permalink to this headline">
@@ -178,8 +178,8 @@
       </div>
      </div>
     </div>
-   </div>
-   <div class="section" id="group-tabs">
+   </section>
+   <section id="group-tabs">
     <h1>
      Group Tabs
      <a class="headerlink" href="#group-tabs" title="Permalink to this headline">
@@ -242,7 +242,7 @@
       </p>
      </div>
     </div>
-   </div>
+   </section>
   </div>
  </div>
 </div>

--- a/tests/test_build/test_conditional_assets_no_tabs2_.html
+++ b/tests/test_build/test_conditional_assets_no_tabs2_.html
@@ -15,7 +15,7 @@
      </li>
     </ul>
    </div>
-   <div class="section" id="fruits">
+   <section id="fruits">
     <h1>
      Fruits
      <a class="headerlink" href="#fruits" title="Permalink to this headline">
@@ -50,8 +50,8 @@
       </p>
      </div>
     </div>
-   </div>
-   <div class="section" id="luminaries">
+   </section>
+   <section id="luminaries">
     <h1>
      Luminaries
      <a class="headerlink" href="#luminaries" title="Permalink to this headline">
@@ -78,8 +78,8 @@
       </p>
      </div>
     </div>
-   </div>
-   <div class="section" id="code-tabs">
+   </section>
+   <section id="code-tabs">
     <h1>
      Code Tabs
      <a class="headerlink" href="#code-tabs" title="Permalink to this headline">
@@ -178,8 +178,8 @@
       </div>
      </div>
     </div>
-   </div>
-   <div class="section" id="group-tabs">
+   </section>
+   <section id="group-tabs">
     <h1>
      Group Tabs
      <a class="headerlink" href="#group-tabs" title="Permalink to this headline">
@@ -242,7 +242,7 @@
       </p>
      </div>
     </div>
-   </div>
+   </section>
   </div>
  </div>
 </div>

--- a/tests/test_build/test_nested_markup.html
+++ b/tests/test_build/test_nested_markup.html
@@ -1,7 +1,7 @@
 <div class="documentwrapper">
  <div class="bodywrapper">
   <div class="body" role="main">
-   <div class="section" id="markup-in-tab-titles">
+   <section id="markup-in-tab-titles">
     <h1>
      Markup in Tab Titles
      <a class="headerlink" href="#markup-in-tab-titles" title="Permalink to this headline">
@@ -206,7 +206,7 @@
       </p>
      </div>
     </div>
-   </div>
+   </section>
   </div>
  </div>
 </div>

